<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlueZQtMediaController</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Manager</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Manager</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QCheckBox" name="checkBox_manager_operational">
                 <property name="text">
                  <string>Operational</string>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_manager_bluetooth_operational">
                 <property name="text">
                  <string>BluetoothOperational</string>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_manager_bluetooth_blocked">
                 <property name="text">
                  <string>BluetoothBlocked</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_discoverable">
                 <property name="text">
                  <string>Discoverable</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Adapters:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="textEdit_adapters"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Devices:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_amanager_scan">
                   <property name="text">
                    <string>Scan</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTextEdit" name="textEdit_devices"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Device</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QComboBox" name="comboBox_devices"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_device">
            <property name="title">
             <string>Device</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_device_name">
                   <property name="text">
                    <string>Name:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_device_address">
                   <property name="text">
                    <string>Address:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_device_type">
                   <property name="text">
                    <string>Type:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_device_ubi">
                   <property name="text">
                    <string>Ubi:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_device_modalias">
                   <property name="text">
                    <string>MODAlias:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_device_uuids">
                     <property name="text">
                      <string>UUIDs:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="comboBox_device_uuids">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QCheckBox" name="checkBox_device_paired">
                   <property name="text">
                    <string>Paired</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_device_pair">
                   <property name="text">
                    <string>Pair</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_device_canclepairing">
                   <property name="text">
                    <string>Cancle Pairing</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_device_connected">
                   <property name="text">
                    <string>Connected</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_device_connect">
                   <property name="text">
                    <string>Connect</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_device_disconnect">
                   <property name="text">
                    <string>Disconnect</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTabWidget" name="tabWidget_2">
                 <property name="tabPosition">
                  <enum>QTabWidget::South</enum>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="tab_4">
                  <attribute name="title">
                   <string>MediaPlayer</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_15">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_14">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_13">
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_8">
                         <item>
                          <widget class="QLabel" name="label_device_mediaplayer_name">
                           <property name="text">
                            <string>Name: -</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QLabel" name="label_device_mediaplayer_status">
                           <property name="text">
                            <string>Status: -</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QLabel" name="label_device_mediaplayer_position">
                           <property name="text">
                            <string>Position: -</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_7">
                         <item>
                          <widget class="QPushButton" name="pushButton_device_mediaplayer_repeat">
                           <property name="enabled">
                            <bool>false</bool>
                           </property>
                           <property name="text">
                            <string>Repeat Off</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="pushButton_device_mediaplayer_shuffle">
                           <property name="enabled">
                            <bool>false</bool>
                           </property>
                           <property name="text">
                            <string>Shuffle Off</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <widget class="QTextEdit" name="textEdit_device_mediaplayer_track">
                         <property name="readOnly">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_6">
                         <item>
                          <widget class="QPushButton" name="pushButton_device_mediaplayer_play">
                           <property name="text">
                            <string>Play</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="pushButton_device_mediaplayer_pause">
                           <property name="text">
                            <string>Pause</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="pushButton_device_mediaplayer_stop">
                           <property name="text">
                            <string>Stop</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="pushButton_device_mediaplayer_next">
                           <property name="text">
                            <string>Next</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="pushButton_device_mediaplayer_previous">
                           <property name="text">
                            <string>Previous</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="pushButton_device_mediaplayer_fastforward">
                           <property name="text">
                            <string>Fast Forward</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="pushButton_device_mediaplayer_rewind">
                           <property name="text">
                            <string>Rewind</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="tab_5">
                  <attribute name="title">
                   <string>...</string>
                  </attribute>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_log_clear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_log">
            <property name="font">
             <font>
              <family>Courier 10 Pitch</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
